scripts/clean.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Deletes temporary and installed files."""
+
+import os
+import re
+import shutil
+
+CURR_DIR = os.path.abspath(os.getcwd())
+OPPIA_TOOLS_DIR = os.path.join(CURR_DIR, '..', 'oppia_tools')
+
+
+def delete_directory_tree(directory_path):
+    """Recursively delete an existing directory tree. Does not do anything if
+    directory does not exists.
+
+    Args:
+        directory_path: str. Directory path to be deleted.
+    """
+    if not os.path.exists(directory_path):
+        return
+    shutil.rmtree(directory_path)
+
+
+def delete_file(filepath):
+    """Delete an existing file. Does not do anything if file does not exists.
+
+    Args:
+        filepath: str. Filepath to be deleted.
+    """
+    if not os.path.isfile(filepath):
+        return
+    os.remove(filepath)
+
+
+delete_directory_tree(OPPIA_TOOLS_DIR)
+delete_directory_tree('node_modules/')
+delete_directory_tree('third_party/')
+delete_directory_tree('build/')
+delete_directory_tree('backend_prod_files/')
+delete_file('.coverage')
+delete_directory_tree('local_compiled_js/')
+delete_directory_tree('local_compiled_js_for_test/')
+delete_file('.viminfo')
+
+for filename in os.listdir(CURR_DIR):
+    if re.search(r'tmpcompiledjs*', filename):
+        delete_directory_tree(filename)
+
+print 'Temporary and installed files deleted'
```

Comment:
Instead of doing this block, do this instead:
```python
def main():
    delete_directory_tree(OPPIA_TOOLS_DIR)
    delete_directory_tree('node_modules/')
    delete_directory_tree('third_party/')
    delete_directory_tree('build/')
    delete_directory_tree('backend_prod_files/')
    delete_file('.coverage')
    delete_directory_tree('local_compiled_js/')
    delete_directory_tree('local_compiled_js_for_test/')
    delete_file('.viminfo')

   for filename in os.listdir(CURR_DIR):
       if re.search(r'tmpcompiledjs*', filename):
           delete_directory_tree(filename)

if __name__ ==  '__main__':
    main()
```

Reply:
Done